src/controllers/likeController.js

```diff
@@ -0,0 +1,66 @@
+import { Article, Like } from '../db/models';
+
+/**
+ *@description This class handles all like requests
+ * @class LikeController
+ */
+class LikeController {
+  /**
+ *@description This function allows a user to like articles
+ * @static
+ * @param {object} req the request body
+ * @param {object} res the response body
+ * @returns {object} res
+ * @memberof LikeController
+ */
+  static async likeArticle(req, res) {
+    const { slug } = req.params;
+    const userId = req.user;
+
+    try {
+      const article = await Article.findOne({
```

Comment:
Can you please move the aspect of finding if an article exists or not to a middleware. This middleware should be reusable.  Also, confirm that it doesn't already exist in our codebase